src/backend/pluginPlan.js

```diff
@@ -40,3 +41,32 @@ export function pluginMirrorPlan(
     await Promise.all(tasks);
   };
 }
+
+export function pluginGraphPlan(
+  {github, discourse, identity}: PluginLoaders,
+  githubToken: ?GithubToken,
+  cache: CacheProvider
+): GraphPlan {
+  return async (project: Project): Promise<Graph> => {
+    const tasks: Promise<Graph>[] = [];
+
+    if (project.discourseServer) {
+      tasks.push(discourse.createGraph(project.discourseServer, cache));
+    }
+
+    if (project.repoIds.length) {
```

Comment:
The reason it needs a token, is because the *name* of the mirror database is based on the GraphQL node ID of the repository. We don't know this ID without querying the remote API.

In the current `master` you can see this here:
https://github.com/sourcecred/sourcecred/blob/dea45fb8c1168addc4667be51d81bc40cc28a292/src/plugins/github/fetchGithubRepo.js#L61-L62